src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallAppsTask.groovy

```diff
@@ -282,11 +286,12 @@ class InstallAppsTask extends AbstractServerTask {
             if (resourceTask.getDestinationDir() != null){
                 looseApp.addOutputDir(archive, resourceTask.getDestinationDir(), "/");
             }
-            looseApp.addManifestFile(archive, siblingProject);
+            File manifestFile = project.sourceSets.main.getOutput().getResourcesDir().getParentFile()
```

Comment:
Should there be a ; at the end for consistency?

Reply:
Doesn't really matter, Groovy is fine with either.